docs/design/node-diagram.md

```diff
@@ -0,0 +1,13 @@
+# Node Diagram
+
+The node diagram designs are depicted below for the Autoware.Core/Universe architecture.
```

Comment:
How about this?

- Use active voice.
  - https://developers.google.com/style/voice?hl=en
- Autoware.Core/Universe -> Autoware Core/Universe
- Remove `below` because it's obvious.
- Remove `the`. (Warned by a tool.)

```suggestion
This page depicts the node diagram designs for Autoware Core/Universe architecture.
```

Reply:
Got it. Can I just commit this suggestion?

Reply:
Sure, please click here.
![image](https://user-images.githubusercontent.com/31987104/157185226-64e33ef8-bdfd-4d73-9ff6-ab192b019b5b.png)